go.mod

```diff
@@ -44,9 +43,33 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0-20170531160350-a96e63847dc3
 	k8s.io/api v0.15.11
-	k8s.io/apiextensions-apiserver v0.0.0-20200318010201-8546efc3bc75 // kubernetes-1.15.11
+	k8s.io/apiextensions-apiserver v0.15.11
 	k8s.io/apimachinery v0.15.11
 	k8s.io/client-go v0.15.11
 	k8s.io/kube-openapi v0.0.0-20190709113604-33be087ad058 // indirect
+	k8s.io/kubernetes v1.15.11
 	k8s.io/utils v0.0.0-20200124190032-861946025e34 // indirect
 )
+
+replace (
```

Comment:
This is required due to the fact that `k8s.io/kubernetes v1.15.11` will require `v0.0.0` of those packages. The require block will enforce the right version.

Reply:
I might take this for a spin, and tweak it, and see what happens :smile: 

I wonder if we can shrink the number of dependencies and still make it work.

Reply:
Took it for a spin. Could not improve on this in any way :frowning: